lib/locales/en.yml

```diff
@@ -41,3 +41,7 @@ en:
     access_for_free: Access for free at openstax.org
   index:
     main: Index
+  exercises:
+    your-turn: Your Turn
```

Comment:
Shouldn't this be recipe side if it's book-specific?

Reply:
Probably 🤔 i’ll move it into my recipe yml